.github/workflows/generate-initial-checksums.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       - name: Move Checksums to Repo
         run: |
           mkdir -p ${{ inputs.committed-checksum-location }}
-          mv ${{ env.OUTPUT_LOCAL_LOCATION }}/* ${{ inputs.committed-checksum-location }}
+          mv ${{ env.OUTPUT_LOCAL_LOCATION }}/checksum/* ${{ inputs.committed-checksum-location }}
```

Comment:
Could you expand a couple of these paths to concrete examples because my head is spinning (it is a Friday afternoon in my defence)

Reply:
so we are `mv /opt/checksum-output/checksum/* ./testing/checksums` if we use the default for `${{ inputs.committed-checksum-location`, where `.` will be the repo, and `/opt/checksum-output` is the extract location for the artifact. 
Inside the artifact would be:
```
/opt/
  `- checksum-output/
     `- checksum
        |- historic-3hr-checksum.json
        `- <maybe other checksums in future>
```

Reply:
Should that be
```
mv /opt/checksum-output/checksum/* ./testing/checksum
```
(note singular not plural `checksum`)?

I note it is correct in the `artefact` location.

https://github.com/ACCESS-NRI/access-om2-configs/blob/main/test/conftest.py#L36

Reply:
Yep, I can change the default to that

Reply:
https://github.com/ACCESS-NRI/access-om2-configs/pull/12